sssom/cliques.py

```diff
@@ -95,7 +112,16 @@ def invert_dict(d: Dict[str, str]) -> Dict[str, str]:
     return invdict
 
 
-def get_src(src, cid):
+def get_src(src: str, cid: str):
+    """Get source.
```

Comment:
A good docstring doesn't just explain what the function does, but also why it does it

Reply:
dont know nothing about clique.py

Reply:
Sure, but it's definitely possible to read this function and check where it's used then figure out what it does. I would estimate this is 2 minutes effort for me since I have a lot of experience doing this (and know all of the keyboard shortcuts in PyCharm 😛) and maybe it will take 5-10 minutes for @hrshdhgd as he gets more used to doing this kind of digital detective work :)